amespahdbpythonsuite/fitted.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, d=None, **keywords):
         Spectrum.__init__(self, d, **keywords)
         self.weights = None
```

Comment:
These ones should probably be _self._weights_ and _self._observation_. Perhaps there should be a separate pull-request to deal with this ...

Reply:
Yes I agree. We should have a separate PR for any changes on the single and double underscores variables, going carefully through the different modules and tests. I will proceed with the small changes in this PR.